### DISCUSSION TITLE
Random order of Views in BaseView (side panel)

### DISCUSSION THREAD

**Discussion by NightSkySK** [aminalaee/sqladmin#642] [Ideas]
Each time once I launch my application I get different order of `ModelViews` in sidebar of `BaseView`
![image](https://github.com/aminalaee/sqladmin/assets/52137446/4d4d08a0-9894-488a-8302-1cab6ff192c5) ![image](https://github.com/aminalaee/sqladmin/assets/52137446/78428f3a-e4c8-4396-8bb8-70d26502a53a)

I use a decorator to `@register_view` dynamically, so I suppose that order is related in which order ModelViews are loaded, but it would be nice feature if we could define an order on the list (not only alphabetically) in wich `ModelViews` are visible on `BaseView` sidebar

**Comment by aminalaee** [comment]
Which version of the SQLAdmin are you using? This was supposed to be fixed in 0.15.1.

**Comment by NightSkySK** [comment]
Yes, I was still back on 0.15.0. It's working great with 0.15.1
